lib/charms/prometheus_k8s/v0/prometheus_scrape.py

```diff
@@ -335,6 +340,81 @@ def _on_scrape_targets_changed(self, event):
 }
 
 
+RELATION_INTERFACE_NAME = "prometheus_scrape"
+
+
+class RelationDirection(Enum):
+    """Relations can be either provided or required by a charm."""
+
+    PROVIDED = 1
+    REQUIRED = 2
+
+
+class NoRelationWithInterfaceFoundError(Exception):
```

Comment:
Compare to `class TooManyRelatedAppsError(ModelError)` in `ops.model`: inherits `ModelError`.